src/prevector.h

```diff
@@ -394,6 +394,21 @@ class prevector {
         fill(ptr, first, last);
     }
 
+    inline void resize_uninitialized(size_type new_size) {
+        // resize_uninitialized change the size of the prevector but does not initialize.
+        // If size < new_size, the added elements must be initialized explicitly after it returns.
```

Comment:
`after it returns.` is unnecessary I think. `must be initialized explicitly`

Reply:
@kallewoof Thanks. fixed.